client/src/components/Bio/Bio.css

```diff
@@ -0,0 +1,29 @@
+.Bio {
+  max-width: 40rem;
+  font-family: "Reem Kufi", sans-serif;
+}
+
+.icons--div {
+  width: 30%;
+  display: flex;
+  justify-content: space-between;
+
+}
+
+.icons--div div {
+  color: #6a4c93;
+}
+
+.Bio h3,
+h2 {
```

Comment:
Please namespace these, otherwise they'll class with styling elsewhere in the app. You can see the part of the readme on BEM for more